package.json

```diff
@@ -13,9 +13,9 @@
         "start:server": "npx @fluidframework/azure-local-service@latest"
     },
     "dependencies": {
-        "@fluidframework/azure-client": "^0.48.1",
-        "@fluidframework/test-client-utils": "^0.48.1",
-        "fluid-framework": "^0.48.1"
+        "@fluidframework/azure-client": "^1.0.1",
```

Comment:
i thought that fluid-framework package is 1.0.1, and the other three (including azure-client) are 1.0.0 per your comment earlier. am i missing something?

Reply:
All of the versions are now on 1.0.1 with the recent local-azure-service issue we needed to resolve.